File: Java/EvenOdd.java
package Java;
import java.util.Scanner;

public class EvenOdd {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number: ");
        int num = sc.nextInt();

        if(num%2==0)
        {
            System.out.println("Number is Even.");
        }
        else{
            System.out.println("Number is Odd.");
        }
    }
}

File: C Programs/EvenOdd.c
#include<stdio.h>

int main()
{
    int num;
    printf("Enter the number: ");
    scanf("%d", &num);

    if(num%2==0)
    {
        printf("Number is Even");
    }
    else
    {
        printf("Number is Odd");
    }

}